packages/mock-app/src/components/Credential/Credential.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { Box } from '@mui/material';
-import { VerifiableCredential } from '@vckit/core-types';
 import { CredentialInfo } from '../CredentialInfo';
 import { CredentialTabs } from '../CredentialTabs';
+import { CredentialComponentProps } from '../../types/common.types';
 
-const Credential = ({ credential }: { credential: VerifiableCredential }) => {
+const Credential = ({ credential, decodedEnvelopedVC }: CredentialComponentProps) => {
   return (
     <Box
       sx={{
```

Comment:
Should we update Storybook and unit tests for these changes to this component?

Reply:
It must be updated but I think in another ticket because now, we use `@vckit/core-types` with the old version, which does not have an Enveloping Proof feature
<img width="857" alt="image" src="https://github.com/user-attachments/assets/ee47ec18-9fd9-4a22-bc5b-b05a2c507158">